common/src/character_controller/mod.rs

```diff
@@ -34,7 +34,7 @@ pub fn run_character_step(
             graph,
             radius: sim_config.character.character_radius,
         },
-        up: graph.get_relative_up(position).unwrap(),
```

Comment:
This comment isn't meaningful. If there's a bug here, please open a separate PR (or issue) and keep this PR focused.